recipes/sstp-client/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sstp-client" %}
+{% set version = "1.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/{{name}}/files/{{name}}/{{name}}-{{version}}.tar.gz
+  sha1: afe80911a5ec033eac583242c642cc5ad00e6ac4
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+    - libevent
+    - openssl
+  run:
+    - libevent
+    - openssl
+
+test:
+  commands:
+    - sstpc --help
+
+about:
+  home: http://sstp-client.sourceforge.net/
+  license: GPL-2.0
+  license_family: GPL
+  license_file: COPYING
```

Comment:
The source tarball already contains the COPYING file, so you can remove it from this recipe

Reply:
I was told that GPL licenses always have to be included in the recipe.